client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -713,7 +717,7 @@ class CalypsoifyIframe extends Component< ComponentProps, State > {
 				{ /* eslint-disable-next-line wpcalypso/jsx-classname-namespace */ }
 				<div className="main main-column calypsoify is-iframe" role="main">
 					{ ! isIframeLoaded && <Placeholder /> }
-					{ isIframeLoaded && (
+					{ isIframeLoaded && this.isEditorTypePost() && (
```

Comment:
IMHO we could simplify this by just checking for the Site Editor:

```suggestion
					{ isIframeLoaded && this.props.editorType !== 'site' && (
```

Reply:
You're right, I updated the PR